main/src/com/google/refine/ProjectMetadata.java

```diff
@@ -373,4 +375,9 @@ public void setAnyField(String metaName, String valueString) {
             logger.error(ExceptionUtils.getFullStackTrace(e));
         }
     }
+
+    @JsonIgnore
+    public void set_lastSave(){
```

Comment:
To follow the existing naming conventions, I would rather name this `setLastSave`.